code/modules/hydroponics/grown/mushrooms.dm

```diff
@@ -292,3 +302,26 @@
 	desc = "A fungus ideal for making antibacterials."
 	icon_state = "angel"
 	color = "#4f4331"
+
+//Shadowshroom
+/obj/item/seeds/glowshroom/shadowshroom
+	name = "pack of shadowshroom mycelium"
+	desc = "This mycelium will grow into something shadowy."
+	icon_state = "mycelium-shadowshroom"
+	species = "shadowshroom"
+	icon_grow = "shadowshroom-grow"
+	icon_dead = "shadowshroom-dead"
+	plantname = "Shadowshrooms"
+	product = /obj/item/weapon/reagent_containers/food/snacks/grown/mushroom/glowshroom/shadowshroom
+	genes = list(/datum/plant_gene/trait/glow/shadow, /datum/plant_gene/trait/plant_type/fungal_metabolism)
+	mutatelist = list()
+	reagents_add = list("radium" = 0.2, "nutriment" = 0.04)
```

Comment:
Oh. Yeah, leave these be, nutriment should still be a *thing* in some plants. If the others don't have plant matter, these shouldn't either.